includes/acl/compression/stream/write_stats.h

```diff
@@ -131,6 +131,53 @@ namespace acl
 		deallocate_type_array(allocator, lossy_local_pose, num_bones);
 	}
 
+	inline void write_decompression_times(Allocator& allocator, const AnimationClip& clip, SJSONObjectWriter& writer,
+		AllocateDecompressionContext allocate_context, DecompressPose decompress_pose, DeallocateDecompressionContext deallocate_context)
+	{
+		uint16_t num_bones = clip.get_num_bones();
+		float clip_duration = clip.get_duration();
+		float sample_rate = float(clip.get_sample_rate());
+		uint32_t num_samples = calculate_num_samples(clip_duration, clip.get_sample_rate());
+
+		void* context = allocate_context(allocator);
+		Transform_32* lossy_pose_transforms = allocate_type_array<Transform_32>(allocator, num_bones);
+
+		{
+			flush_data_cache(allocator);
+
+			ScopeProfiler timer;
+
+			for (uint32_t sample_index = 0; sample_index < num_samples; ++sample_index)
+			{
+				float sample_time = min(float(sample_index) / sample_rate, clip_duration);
+				decompress_pose(context, sample_time, lossy_pose_transforms, num_bones);
```

Comment:
the cache should be cleared before we decompress each pose to simulate a cold cache.